fs/mount/fs_procfs_mount.c

```diff
@@ -248,6 +248,10 @@ static int blocks_entry(FAR const char *mountpoint,
 
   if (!info->header)
     {
+#if defined(CONFIG_HAVE_LONG_LONG) && !defined(CONFIG_LIBC_LONG_LONG)
+      finfo("long long is enabled, but libc does not support it;\n"
```

Comment:
it's strange to print log here, or you can use static_assert instead.

Reply:
finfo is debugging output, activated with CONFIG_DEBUG_FS_INFO, and is directed at developers early in prototyping, who are trying to work with **large** filesystems without enabling LONG_LONG.

Reply:
Yes, but can be a compile time `#warning` instead of console printf

Reply:
@mlyszczek proposed it in #5253. I can change this  (no-op on production builds) to a preprocessor warning. We cannot know predict at compile time what size sdcards will be inserted at run-time. Oh well, that should be handled by the next fwarn()'s.

Reply:
There are many places which call [s]printf with [u]int64_t, so it isn't good to check it only here or spread the check around the whole code base. The good fix is do the check inside the core of printf like https://github.com/apache/incubator-nuttx/pull/6613.

Reply:
Yes, that's better, if you are sure about refactoring core printf.
I will drop this proposed check from fs/mount then. And always pass uint64_t.